examples/advanced/main.tf

```diff
@@ -66,7 +94,7 @@ module "slz_baremetal" {
   prefix                = var.prefix
   profile               = var.profile
   image_id              = data.ibm_is_image.slz_vsi_image.id
-  subnet_ids            = [for subnet in module.slz_vpc.subnet_zone_list : subnet.id if subnet.zone == "${var.region}-1"]
+  subnet_ids            = [for subnet in module.slz_vpc.subnet_zone_list : subnet.id if subnet.zone == "${var.region}-1"][0]
```

Comment:
This change of adding the index 0 of the array is changing this datatype from "array of strings" to just a simple "string", which is the wrong data type for the module input.